compliance/Azure_long_stopped_instances/long_stopped_instances_azure.pt

```diff
@@ -0,0 +1,210 @@
+name "Azure long stopped instances"
```

Comment:
The path for this file should be in the compliance/azure directory, the file name is OK as it.  please rename the directory Azure_long_stopped_instances to azure.

Reply:
Done